x/profiles/keeper/handler.go

```diff
@@ -140,3 +149,88 @@ func handleMsgDeleteProfile(ctx sdk.Context, keeper Keeper, msg types.MsgDeleteP
 
 	return &result, nil
 }
+
+// handleMsgRequestDTagTransfer handles the request of a dTag transfer
+func handleMsgRequestDTagTransfer(ctx sdk.Context, keeper Keeper, msg types.MsgRequestDTagTransfer) (*sdk.Result, error) {
+	transferRequest := types.NewDTagTransferRequest(msg.CurrentOwner, msg.ReceivingUser)
+
+	if err := keeper.SaveDTagTransferRequest(ctx, transferRequest); err != nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, err.Error())
+	}
+
+	ctx.EventManager().EmitEvent(sdk.NewEvent(
+		types.EventTypeDTagTransferRequest,
+		sdk.NewAttribute(types.AttributeCurrentOwner, transferRequest.CurrentOwner.String()),
+		sdk.NewAttribute(types.AttributeReceivingUser, transferRequest.ReceivingUser.String()),
```

Comment:
Proably we can add an attribute to tell which DTag the user wants to trade?

Reply:
This means using `keeper.GetDtagFromAddress(ctx, address)` function just to retrieve the DTag.
Is it worth it?